prac_07/project_management.py

```diff
@@ -0,0 +1,139 @@
+"""
+Project Management Program
+Estimate: 45 minutes
+Actual: 125 minutes
+"""
+import datetime
+from project import Project
+
+MENU = """- (L)oad projects  
+- (S)ave projects  
+- (D)isplay projects  
+- (F)ilter projects by date
+- (A)dd new project  
+- (U)pdate project
+- (Q)uit"""
+
+
+def main():
+    """Process project information based on user's choice."""
+    filename = input(">>> ")
```

Comment:
I think it's better to make the file constant rather than input the filename everytime you run the code.